assignments/005-slurm-and-bwa-mem2/index2.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#SBATCH --cpus-per-task=1
+#SBATCH --time=2:00:00
+#SBATCH --mem=3G
+#SBATCH --output=fastp_trim-%j.out
+#SBATCH --error=fastp_trim-%j.err
+
+
+# make a directory for log output if it does not already exist
+DIR=results/log/bwa_index2
+mkdir -p $DIR
+
+
+# indexing using bwa-mem2
+./../miniforge3/bin/bwa-mem2 index $1> $DIR/log.txt 2>&1
```

Comment:
It would be better to activate the environment like this:

```
source ~/.bashrc
conda activate bwa-mem2
```

Then you can just call `bwa-mem2` and it will be in your PATH.  
```